lib/components/template/TemplateRenderer.vue

```diff
@@ -1,24 +1,39 @@
 <template lang="pug">
-iframe(v-if="isSafari" :src="pdfBlob")
-object(v-else :data="pdfBlob" type="application/pdf")
-//- embed(:src="pdfBlob" width="100%" height="100%" name="plugin" id="plugin" type="application/pdf")
+div(v-if="exportFormat == 'pdf'")
+  iframe(v-if="isSafari" :src="pdfBlob")
+  object(v-else :data="pdfBlob" type="application/pdf")
+  //- embed(:src="pdfBlob" width="100%" height="100%" name="plugin" id="plugin" type="application/pdf")
+
+div(v-else-if="exportFormat == 'jpg'")
+  img(:src="jpgData")
+
+div(v-else-if="exportFormat == 'png'")
+  img(:src="pngData")
+
+div(v-else-if="exportFormat == 'svg'")
+  div(v-html="svgString")
```

Comment:
PNG and JPG come back as Data URIs and get bound as an image source
SVG is perfect as a string, I'm throwing that directly into the browser via the `v-html` directive.

Inspect these elements and you'll see they're just regular old dom elements, so all right-clicking and saving and whatnot work as expected.

Reply:
Thank you so much!